packages/ffe-context-message-react/src/ContextSuccessMessage.js

```diff
@@ -11,19 +14,28 @@ const ContextSuccessMessage = props => {
 
     return (
         <ContextMessage
-            {...props}
+            {...rest}
+            locale={locale}
             messageType="success"
-            aria-label="Suksessmelding"
```

Comment:
Her er jeg ikke sikker på om jeg er enig at aria-label skal fjernes. Fordi når den ligger på group så leses den opp hele boksen istedenfor at brukerne må navigere seg inn til ikonet.

Reply:
Så ikke noen exempel på alert og aria label. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role

Er ikke risken med en arai-label på alerten att den gir blaffen i texten i selva alreten og bruker selva aria labeln som innehold? Har ikke testet selv. Enig i aria-label gir mening på "role=group"

Reply:
Snakket med Sindre og båda blir lest op så da er dette greit.

Reply:
Her er vad jag var redd for iaf 
https://www.aditus.io/aria/aria-label/

Knappen får ikke "send email send". Alltså send i knappen blir spist op.

Reply:
![image](https://github.com/SpareBank1/designsystem/assets/2248579/f4bb368c-fec9-4f7a-bf8a-bb8a0390c1dd)
"Ignored"